contributed_definitions/NXregion.nxdl.xml

```diff
@@ -151,12 +151,11 @@
 	</field>
 	<field name="scale" type="NX_NUMBER">
 		<doc>
-			An optional field to define a divisor for scaling of reduced data.
-			This can also be **O** + **R** in length so that each dimension's region can be
-			divided by a different value. For example, in a downsampled sum, it can reduce
-			the maximum values to fit in the domain of the result data type. In an image that
-			is downsampled by summing 2x2 blocks, using :math:`\mathrm{scale}=4' allows the result
-			to fit in the same integer type dataset as the parent dataset.
+			An optional field to define a divisor for scaling of reduced data. For example, in a
+			downsampled sum, it can reduce the maximum values to fit in the domain of the result
+			data type. In an image that is downsampled by summing 2x2 blocks, using
+			:math:`\mathrm{scale}=4' allows the result to fit in the same integer type dataset as
+			the parent dataset.
 			If omitted then defined as an array of ones
```

Comment:
I believe this should be: `If omitted the scale is assumed to be 1`. You change the scale to be a scalar right?